src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -281,18 +288,19 @@ public void clickViewPhotoLink(int panelBodyIndex, String urlRegex) throws Excep
                                              .get(0);
         JavascriptExecutor jsExecutor = (JavascriptExecutor) browser.driver;
         jsExecutor.executeScript("document.getElementById('" + idOfPanelBody + "')"
-                + ".getElementsByClassName('profile-pic-icon-click')[0]"
-                + ".getElementsByTagName('a')[0].click();");
+                                 + ".getElementsByClassName('profile-pic-icon-click')[0]"
+                                 + ".getElementsByTagName('a')[0].click();");
         Actions actions = new Actions(browser.driver);
-        
+
         actions.moveToElement(photoCell).build().perform();
         waitForElementToAppear(By.cssSelector(".popover-content > img"));
-        
-        AssertHelper.assertContainsRegex(urlRegex, 
-                browser.driver.findElements(By.cssSelector(".popover-content > img"))
-                              .get(browser.driver.findElements(By.cssSelector(".popover-content > img")).size() - 1)
-                              .getAttribute("src"));
-        
+
+        AssertHelper.assertContainsRegex(urlRegex,
+                                         browser.driver.findElements(By.cssSelector(".popover-content > img"))
+                                                       .get(browser.driver.findElements(By.cssSelector(".popover-content > img"))
+                                                                                          .size() - 1)
+                                                       .getAttribute("src"));
```

Comment:
The new one is not more readable than the original. See if there is a better alternative.